functions/two_sum.py

```diff
@@ -1,2 +1,7 @@
-def twoSum(n, t):
-    pass
+def two_sum(n, t):
+    for i in range(len(n)):
+        for j in range(i + 1, len(n)):
```

Comment:
can you do better than a nested loop for time complexity?

Reply:
Yes, I thought about this but it's too much work for me right now...